src/app/modules/navigator/test-exec-navigator.component.ts

```diff
@@ -16,6 +16,14 @@ import { idPrefix } from '../module-constants';
 
 export const EMPTY_TREE: TreeNode = { name: '<empty>', root: null, children: [] };
 
+export type UITestRunStatus = 'running'| 'successful' | 'failure';
+export interface UITestRun {
+  class: UITestRunStatus;   // css class to use for display
```

Comment:
you could probably get rid of the comments with slightly more expressive (but also longer) identifiers